### PR TITLE
Changed some paths to the github where the 404 error was

### PR DESCRIPTION
**Pull request by kogeletey**

### CODE REVIEW

File: docs/fxa-engineering/fxa-metrics.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ The tables are visible in Redash, under the “FxA Activity Metrics” data sour
 
 Our Amplitude events are sent to a GCP pubsub queue from stackdriver, this stuff is managed by jbuck. We have a node script that listens to the queue and then sends the events to Amplitude:
 
-* https://github.com/mozilla/fxa-amplitude-send/blob/master/pubsub/index.js
+* https://github.com/mozilla/fxa-amplitude-send/blob/main/pubsub/index.js
```
Comment (clouserw) [inline docs/fxa-engineering/fxa-metrics.md:130]: This is still a 404.  Let's just link to the top level repository: https://github.com/mozilla/fxa-amplitude-send

File: docs/fxa-engineering/fxa-metrics.md [hunk 2]
```diff
@@ -176,7 +176,7 @@ You don’t need to do this, it’s all set up to work automatically.
 # What other docs are there?
 * https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/metrics-events.md
 * https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/docs/client-metrics.md
-* https://github.com/mozilla/application-services/blob/master/docs/product-portal/accounts/metrics.md
+* https://github.com/mozilla/application-services/blob/main/docs/metrics.md
```
Comment (clouserw) [inline docs/fxa-engineering/fxa-metrics.md:179]: This is a 404 too.  Looks like this should be https://github.com/mozilla/application-services/tree/main/docs/metrics

File: docs/fxa-engineering/fxa-metrics.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ The Heka filter does some processing on the events and then writes them once-per
 The CSV files are then processed by python scripts in the `mozilla/fxa-activity-metrics` repo, running on the fxa-admin EC2 instance (managed by jrgm and jbuck):
 
 * https://github.com/mozilla/fxa-activity-metrics/
-* https://github.com/mozilla-services/cloudops-deployment/blob/master/projects/fxa/puppet/modules/fxa_admin/manifests/init.pp
+* https://github.com/mozilla-services/cloudops-deployment/blob/main/projects/fxa/puppet/modules/fxa_admin/manifests/init.pp
```
Comment (clouserw) [inline docs/fxa-engineering/fxa-metrics.md:103]: The old URL was correct here

**Review by clouserw — CHANGES_REQUESTED** [review]
Thanks for the patch.  I left some comments

File: docs/fxa-engineering/fxa-metrics.md (hunk 3 above)
Comment (kogeletey) [reply docs/fxa-engineering/fxa-metrics.md:103]: But tell me, this is a closed repository, I understand correctly. I just don't have such a repository.

Comment (clouserw) [reply docs/fxa-engineering/fxa-metrics.md]: Yep, I know it's private.  I'm just saying the old URL is still the correct one.

**Review by clouserw — APPROVED** [review]
Thanks!

**Pull request merged**